workflows/toast_sim_ground.py

```diff
@@ -104,6 +109,21 @@ def parse_config(operators, templates, comm):
         help="Only sample the provided focalplane pixels",
     )
 
+    parser.add_argument(
+        "--fknee",
```

Comment:
Would it not be better to generate a focalplane file that has this value?  The [included script](https://github.com/hpc4cmb/toast/blob/7a0a7b29d5f0e886f7ed75a389c43a09bd230f70/src/toast/scripts/toast_fake_focalplane.py#L85) already does that.  Otherwise it seems like we risk adding more and more options here to override specific things in the inputs, rather than just changing the inputs.

For example, we already have other options (`--thinfp` and `--sample_rate`) which are also part of the focalplane and should arguably be removed (i.e. just use a different focalplane).  However, I realize that might break existing scripts.

Reply:
I removed `--fknee` but I feel both `--thinfp` and `sample_rate` have their place. They allow rapid descoping of the simulation for testing purposes.

Reply:
Sounds good, thanks for the consideration.